sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -909,4 +910,21 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-34152: global temp view's identifier should be correctly stored") {
```

Comment:
can we move it to the new `GlobalTempViewTestSuite`? we should slowly move all the view tests to `SQLViewTestSuite` framework.

Reply:
+1 for @cloud-fan 's suggestion.